### PR TITLE
Edu benefits copy updates

### PR DESCRIPTION
**Pull request by jbalboni**
These are copy updates (based on the copy in the edu-benefits-team repo) that were made for the stakeholder review.

This is mainly the benefits selection page. There are also some updates on the military service page and the direct deposit page.

I left out the "I don't know" option on the benefits selection page since there are some questions around that.

screen shot 2016-09-22 at 5 49 14 pm
screen shot 2016-09-22 at 5 57 42 pm
screen shot 2016-09-22 at 5 58 11 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@akainic Can you take a look at this one?

File: src/sass/modules/_m-form-elements.scss [hunk 1]
```diff
@@ -46,6 +46,16 @@ button.form-button-disabled {
   label {
     margin-top: 1em;
   }
+  > input[type="radio"] + label::before {
```
Comment (jbalboni) [inline src/sass/modules/_m-form-elements.scss]: This is to line up the text after radio buttons like we do for checkboxes.

File: src/sass/edu-benefits.scss [hunk 2]
```diff
@@ -56,4 +56,14 @@ legend {
   label:first-child {
     margin-top: 0;
   }
-}
\ No newline at end of file
+}
+
+.edu-benefits-list {
```
Comment (jbalboni) [inline src/sass/edu-benefits.scss]: This is the uswds standard list styling.

**Comment by akainic** [comment]
Looks great! One question I have is about the styling on the benefits selection page. Why have one part of the conditions be in a blue callout box and the other not? If it's to visually differentiate them, can we not have both in their own callout box, with space between? Or not use callout boxes at all and use some kind of other visual marker to differentiate these parts?

**Comment by jbalboni** [comment]
@akainic That's a good question. I think @cweber and @goldenmeanie wanted to work more on how all that copy is structured.

**Comment by cweber** [comment]
Yep. We're going to work on both the copy and on how we're visually handling nested content. I expect this section will be influenced by both of those things.

**Pull request merged**